packages/ui/src/Hyperlink.tsx

```diff
@@ -37,7 +37,7 @@ const linkifyLib = require("linkify-it")();
 
 const {OS} = Platform;
 
```

Comment:
Can we add a comment describing the difference between using Hyperlink and Link?